CMakeLists.txt

```diff
@@ -197,11 +197,11 @@ include(CMakeDependentOption)
 CMAKE_DEPENDENT_OPTION(WITH_ZBD "Enable libzbd bluestore backend" OFF
   "WITH_BLUESTORE" OFF)
 if(WITH_ZBD)
-  find_package(zbd)
+  find_package(zbd REQUIRED)
   set(HAVE_LIBZBD ${ZBD_FOUND})
 endif()
 
-CMAKE_DEPENDENT_OPTION(WITH_LIBURING "Enable io_uring bluestore backend" OFF
+CMAKE_DEPENDENT_OPTION(WITH_LIBURING "Enable io_uring bluestore backend" ON
```

Comment:
does this mean we'd be building liburing by default? if so, how much build time does that add, and is it turned off if the kernel doesn't support io_uring?

Reply:
> does this mean we'd be building liburing by default?

yes, as the commit message reads. it builds liburing by default.

> if so, how much build time does that add

around 3 minutes top depending on the cache hit rate. currently, it takes 20 to 31 minutes to build and test the ceph tree. as the ccache's cache hit rate is relatively high. if the cache of ccache is still valid, it takes 0 build time.

> and is it turned off if the kernel doesn't support io_uring?

this iouring backend is disabled unless specified otherwise. that's why i added `--bluestore-io-uring` to vstart.sh. and yes, it's disabled at runtime if kernel does not support io_uring. see https://github.com/ceph/ceph/blob/99a36731f000283b784d87dc96e75a7f290a180b/src/blk/kernel/KernelDevice.cc#L72-L83